common/containers/Tabs/Swap/components/LiteSend/LiteSend.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import { OfflineAwareUnlockHeader } from 'components';
+import { OnlyUnlocked } from 'components/renderCbs';
+import { Aux } from 'components/ui';
+import { Fields } from './Fields';
+import { configureLiteSend, TConfigureLiteSend } from 'actions/swap';
+import { connect } from 'react-redux';
+import { AppState } from 'reducers';
+import { shouldDisplayLiteSend } from 'selectors/swap';
+
+interface DispatchProps {
+  configureLiteSend: TConfigureLiteSend;
+}
+
+interface StateProps {
+  shouldDisplay: boolean;
+}
+
+type Props = StateProps & DispatchProps;
+class LiteSendClass extends Component<Props> {
+  public componentDidMount() {
+    this.props.configureLiteSend();
+  }
+
+  public render() {
+    if (!this.props.shouldDisplay) {
+      return null;
+    }
+    return (
+      <Aux>
+        <OfflineAwareUnlockHeader allowReadOnly={false} />
```

Comment:
I think we want to use WalletDecrypt instead. This adds a title that shouldn't be there, and you wouldn't be offline mid-swap anyway. The only thing we'd need to add is a button to change wallets into the Fields component.